src/bindings/scala/api/src/main/scala/com/ctc/omega_edit/SessionImpl.scala

```diff
@@ -77,13 +77,13 @@ private[omega_edit] class SessionImpl(p: Pointer, i: FFI) extends Session {
     Edit(i.omega_edit_delete(p, offset, len))
 
   def insert(b: Array[Byte], offset: Long): Result =
-    Edit(i.omega_edit_insert_bytes(p, offset, b, 0))
-
+    Edit(i.omega_edit_insert_bytes(p, offset, b, b.length.toLong))
```

Comment:
Definitely a good change here.  Sending 0 in as the length forces the Ωedit C code to use strlen on it, which is not good here.  It's fine for testing character strings, but not fine for real code.  It's there as a convenience for writing oodles of test code, but not meant for production code.  I should add a warning in the API documentation about this.  Sorry if that was confusing.